pkg/function/delete_csi_snapshot_content.go

```diff
@@ -81,6 +89,10 @@ func (*deleteCSISnapshotContentFunc) Arguments() []string {
 	}
 }
 
+func (c *deleteCSISnapshotContentFunc) ExecutionProgress() (crv1alpha1.PhaseProgress, error) {
+	return crv1alpha1.PhaseProgress{ProgressPercent: string(c.progressPercent)}, nil
```

Comment:
Do we not want to set the time here?

Reply:
and at some other places?

Reply:
Yes, missed at some places. I will correct that, thanks